tests/desktop/test_complete_themes.py

```diff
@@ -41,12 +41,12 @@ def test_that_complete_themes_can_be_sorted_by_updated_date(self, mozwebqa):
         complete_themes_page.click_sort_by("recently updated")
         addons = complete_themes_page.addon_names
         addons_set = set(addons)
-        Assert.equal(len(addons), len(addons_set), "There are duplicates in the names")
+        assert len(addons) == len(addons_set), 'There are duplicates in the names'
         updated_dates = complete_themes_page.addon_updated_dates
-        Assert.is_sorted_descending(updated_dates)
+        assert updated_dates.sort(reversed=True)
```

Comment:
This is not doing what you expect. We need to compare `updated_dates` to `updated_dates.sort(reversed=True)`